pollux/lib/sql-doobie/src/main/resources/sql/pollux/V16__revocation_status_lists_table_and_columns.sql

```diff
@@ -16,7 +16,11 @@ CREATE TABLE public.credential_status_lists
     updated_at   TIMESTAMP WITH TIME ZONE                   NOT NULL default now()
 );
 
-CREATE INDEX credential_status_lists_wallet_id_index ON public.credential_revocation_status_lists (wallet_id);
+CREATE INDEX credential_status_lists_wallet_id_index ON public.credential_status_lists (wallet_id);
+
+ALTER TABLE public.issue_credential_records
+    ALTER COLUMN id TYPE UUID USING id::uuid;
```

Comment:
can you ask @FabioPinheiro or @bvoiturier why the id field in the `issue_credentil_record` is not uuid
as far as I remember, it should be a string because of some protocol constraints

Reply:
I think that id is a copy of the DID Comm message id which is not necessarily a UUID